### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Tabris2k** [Supamiu/Lostark-helper#61]
Hi. I love your application so far. Tried to contact you to make a suggestion, but this is the only way I found.

It would be neat to add a secction with an "Island Checklist" for Island Souls and Island Mokokos. So we can quickly check which ones we have instead of going to the minimap and hover the mouse from island to island to see which ones we have completed.

I don´t have any idea about programming, so I don't know how difficult or time-consuming this would be, but please take it into consideration.

Thanks for listening and please keep improving, you're doing a wonderful job.

**Comment by atheistium** [comment]
Yeah a "to do" checklist of collectables and rapport would be really nice. 

https://mokoko.info/guides/rapport is nice for this but as there's no login you you can easily lose progress. 

Having a all-in-one website would be amazing.
